pkg/controller/telemetry/metric.go

```diff
@@ -78,30 +78,41 @@ type serviceMetricInfo struct {
 	ServiceConnFailed        float64
 }
 
+// connectionDataV4 read from ebpf ringbuf and padding with `_`
 type connectionDataV4 struct {
 	SrcAddr        uint32
 	DstAddr        uint32
 	SrcPort        uint16
 	DstPort        uint16
-	RedundantData  [6]uint32
+	_              [6]uint32
+	OriginalAddr   uint32
+	OriginalPort   uint16
+	_              [3]uint32
+	_              uint16
```

Comment:
Is this because of byte aliging, we should reorder the fields then.

Reply:
yes this aims for byte aligning, I will try to reorder it

Reply:
Because the length of data stored in ringbuf for ipv4 and ipv6 is the same. In order to be able to parse the data properly, blank fields need to be filled in. 
But other fields need to be reordered and byte-aligned manually